pkgs/top-level/all-packages.nix

```diff
@@ -21216,10 +21216,15 @@ with pkgs;
     asio = asio_1_10;
   };
 
-  mariadb = callPackage ../servers/sql/mariadb {
+  inherit (callPackage ../servers/sql/mariadb {
     inherit (darwin) cctools;
     inherit (darwin.apple_sdk.frameworks) CoreServices;
-  };
+  })
+    mariadb_104
```

Comment:
I've noticed more of a pattern in nixpkgs that would have us choose `mariadb_10_4`... but maybe I'm off base?

Reply:
No idea which way we normally do this, tbh. Do versionOlder and versionAtLeast understand versions with _ in them?

Reply:
OK. As is should work fine then :+1: